clang/include/clang/Basic/TokenKinds.def

```diff
@@ -431,6 +435,10 @@ CXX20_KEYWORD(constinit             , 0)
 CXX20_KEYWORD(concept               , 0)
 CXX20_KEYWORD(requires              , 0)
 
+// C++ contracts keywords
+CONTRACTS_KEYWORD(contract_assert)
+ALIAS("__contract_assert"        , contract_assert      , KEYCXX11)
```

Comment:
Do we want to bother? We pickerd a terrible keyword because it was not used, and if you have a macro called `__contract_assert`, i sort of don't care